articles/native-platforms/android/00-introduction.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Introduction
+description: Short Introduction to the Auth0 Android Quickstarts.
+---
+
+<%= include('../../_includes/_github', { link: 'https://github.com/inaka/auth0-android-sample/tree/00-introduction', }) %>
+
+This is a short introduction to the Auth0 Android Quickstarts.
```

Comment:
@tenmaster Replace this text with the one from login: "This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your android apps with Auth0."